React/src/pages/orderStages/OrderStageCard.tsx

```diff
@@ -0,0 +1,454 @@
+                    </LocalizationProvider>
+                </Grid>
+
+                {isDisplayingMode ? (
+                    <Grid item xs={6}>
+                        <LocalizationProvider dateAdapter={AdapterDayjs}>
+                            <DatePicker
+                                sx={{
+                                    width: '100%',
+                                    '& .MuiInputBase-input.Mui-disabled': {
+                                        WebkitTextFillColor: '#000000',
+                                    },
+                                }}
+                                label="Data zakończenia"
+                                disabled={isDisplayingMode}
+                                onChange={(data) => {
+                                    // const formattedDate = dayjs(data).format('YYYY-MM-DDTHH:mm:ss.SSS')
+                                }}
+                            />
+                        </LocalizationProvider>
+                    </Grid>
+                ) : null}
+                <Grid item xs={6}>
+                    <LocalizationProvider dateAdapter={AdapterDayjs}>
+                        <TimePicker
+                            sx={{
+                                width: '100%',
+                                '& .MuiInputBase-input.Mui-disabled': {
+                                    WebkitTextFillColor: '#000000',
+                                },
+                            }}
+                            disabled={isDisplayingMode}
+                            label="Planowana godzina zakończenia"
+                            format={'HH:mm:ss'}
+                            value={plannedFinishHour}
+                            onChange={(data) => {
+                                setPlannedFinishHour(data)
+                            }}
+
+                            // slotProps={{
+                            //     textField: {
+                            //     //   fullWidth: fullWidth,
+                            //     //   variant: 'outlined',
+                            //       error={formik.touched.name && Boolean(formik.errors.plannedEndDate)}
+                            //       helperText={formik.touched.plannedEndDate && formik.errors.plannedEndDate}
+                            //     },
+                            //   }}
+                        />
+                    </LocalizationProvider>
+                </Grid>
+            </Grid>
+        )
+    }
+
+    return (
+        <Card id={index ? index.toString() : ''} sx={{ margin: 'auto', marginTop: '20px', border: '1px solid' }}>
+            <CardActions disableSpacing>
+                <IconButton aria-label="share">
+                    <AssignmentIcon />
+                </IconButton>
+                <Typography variant="h5" color="text.secondary">
+                    {stage ? <Typography>Etap stage.order zlecenia</Typography> : <Typography>Dodaj etap</Typography>}
```

Comment:
```suggestion
                    {stage ? <Typography>Etap {stage.name} zlecenia</Typography> : <Typography>Dodaj etap</Typography>}
```
A tutaj trzeba poprwić, żeby nazwa się wyświetlała